playwright/utils/sierra-api-utils.ts

```diff
@@ -0,0 +1,53 @@
+export interface SierraTokenResponse {
+  access_token: string;
+  token_type: string;
+  expires_in: number;
+}
+
+const SIERRA_API_BASE_URL_QA = "https://nypl-sierra-test.nypl.org";
```

Comment:
If are using an environment variable now, we'll need to pull in in here like this: `const SIERRA_API_BASE_URL_QA = process.env.SIERRA_API_BASE_URL_QA` like we do with the `basicAuth` variable below. 

And similarly, we'll want to add this to `.env.local` (and `.env`) for running locally